test/rubygems/test_gem_ext_cargo_builder.rb

```diff
@@ -58,7 +58,7 @@ def test_build_cdylib
 
     output = output.join "\n"
 
-    bundle = File.join(@dest_path, "release/rust_ruby_example.#{RbConfig::CONFIG['DLEXT']}")
+    bundle = Dir[File.join(@dest_path, "**/release/rust_ruby_example.#{RbConfig::CONFIG['DLEXT']}")].first
```

Comment:
Any info on this change?

Reply:
Good catch, it was accidental. I just reverted it.